ecs-task-definition.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
will need to be changed once we get specific aws roles but adding as a place holder for now